packages/deepsignal/test/index.test.tsx

```diff
@@ -1,11 +1,9 @@
-import "@preact/signals";
+// @ts-ignore
```

Comment:
Same here with the `@ts-ignore` comment.

Reply:
Oh, that one I know 😄

Added: https://github.com/luisherranz/deepsignal/pull/11/commits/79336ad829183465408cd8fd87e160c394760fbb